storage/innobase/xtrabackup/test/t/bug1254227.sh

```diff
@@ -25,6 +25,10 @@ XA END 'xatrx';
 XA PREPARE 'xatrx';
 EOF
 
+# Let the client complete the above set of statements
```

Comment:
do we need to run this in another thread. Can't we just run this sql in before taking backup?

Reply:
ack. thanks for clarification.